### PR TITLE
Update endpoint URL in OTLP-DD ingestion doc

### PR DESCRIPTION
**Pull request by rupalivohra**
### What does this PR do?

Updates the grpc endpoint example in the docs.

### Motivation

I think it's a typo.

### Additional Notes

I haven't tested this out yet.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by drichards-87** [comment]
Thanks for submitting this PR! I've created DOCS-5168 for the review.

**Comment by kayayarai** [comment]
@gbbr Hi Gabriel, can you verify whether this suggested change around the grpc URL is correct, or whether the fact that it's pointing at 4317 is what indicates it's grpc not http?

File: content/en/opentelemetry/otlp_ingest_in_the_agent.md [hunk 1]
```diff
@@ -193,7 +193,7 @@ There are many other environment variables and settings supported in the Datadog
         fieldRef:
           fieldPath: status.hostIP
     - name: OTEL_EXPORTER_OTLP_ENDPOINT
-      value: "http://$(HOST_IP):4317" # sends to gRPC receiver on port 4317
+      value: "grpc://$(HOST_IP):4317" # sends to gRPC receiver on port 4317
```
Comment (gbbr) [inline content/en/opentelemetry/otlp_ingest_in_the_agent.md:196]: Our documentation is correct. gRPC is HTTP. The URL doesn't need to change. Why do you think this change is necessary? Please take a look at https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#configuration-options. There is an example there.

**Review by gbbr — CHANGES_REQUESTED** [review]
I'd suggest we don't make this change.

**Pull request closed without merging**